src/cr/cube/cube.py

```diff
@@ -506,36 +503,35 @@ def _measures(self):
 
     @lazyproperty
     def _numeric_array_dimension(self):
-        """Column dimension object according to the mean subvariables."""
+        """Row dimension object according to the mean subvariables."""
         if not self._mean_subvariables:
             return None
         subrefs = self._mean_references.get("subreferences", [])
-        column_dimension = {
+        row_dimension = {
             "references": {
                 "alias": self._mean_references.get("alias", "mean"),
                 "name": self._mean_references.get("name", "mean"),
             },
             "type": {"elements": [], "class": "enum", "subtype": {"class": "num_arr"}},
         }
-        # ---In case of numeric arrays the column dimension should contains additional
+        # ---In case of numeric arrays the row dimension should contains additional
         # ---information related to the subreferences for each subvariable of the
         # ---array.
         for i, _ in enumerate(self._mean_subvariables):
-            # ---The column dimensions elements must be expanded with the alias and the
+            # ---The row dimensions elements must be expanded with the alias and the
             # ---name of the numeric array mean measure subreferences.
-            column_dimension["type"].get("elements", []).append(
+            row_dimension["type"].get("elements", []).append(
                 {
                     "id": i,
                     "value": {
                         "references": {
                             "alias": subrefs[i].get("alias") if subrefs else None,
                             "name": subrefs[i].get("name") if subrefs else None,
-                        },
```

Comment:
We should still leave this here, it conforms to all the other subvars. We're not gonna use it though, but we should keep it...